.github/workflows/third-party-library-changes.yml

```diff
@@ -0,0 +1,63 @@
+name: Third Party Library Changes
+
+on:
+  pull_request:
+    paths:
+      - 'src/wp-includes/ID3/**'
+      - 'src/wp-includes/IXR/**'
+      - 'src/wp-includes/PHPMailer/**'
+      - 'src/wp-includes/Requests/**'
+      - 'src/wp-includes/SimplePie/**'
+      - 'src/wp-includes/sodium_compat/**'
+      - 'src/wp-includes/Text/**'
+
+# Disable permissions for all available scopes by default.
+# Any needed permissions should be configured at the job level.
+permissions: {}
+
+jobs:
+  third-party-changed-files:
+    runs-on: ubuntu-latest
+    name: Get third party library modified files
+    permissions:
+      issues: write
+      pull-requests: write
+    if: ${{ github.repository == 'johnbillion/wordpress-develop' }}
+    steps:
+      - name: Leave a comment about third party libraries
```

Comment:
Wondering if we should follow the reusable workflow pattern that is now everywhere to make updating this easier in the future.